src/geo/transform.js

```diff
@@ -1355,11 +1361,9 @@ class Transform {
 
         this._constraining = true;
 
-        let minY = -90;
-        let maxY = 90;
-        let minX = -180;
-        let maxX = 180;
-        let sy, sx, x2, y2;
+        let minY = Infinity;
+        let maxY = -Infinity;
+        let minX, maxX, sy, sx, y2;
```

Comment:
It looks to me like minX and maxX could be accessed uninitialized. Do they need initialization?

```suggestion
        let minX = Infinity;
        let maxX = -Infinity;
        let sy, sx, y2;
```